src/components/Header/Header.js

```diff
@@ -0,0 +1,12 @@
+import { Link } from "gatsby"
+import React from "react"
+import logoWhite from "../../images/incentro_logo_white.png"
+
+const Header = () => (
+  <header>
+    <img src= {logoWhite} alt="logo_white" style={{width: "200px", height: "35px", verticalAlign: "middle"}}/>
+    <p style={{color: "white", fontSize: "16px"}}>CV Creator 2.0</p>
+  </header>
+)
+
+export default Header
```

Comment:
I think you need 1 new line at the end of each document. Usually the editor does that automatically.